it/mail/chrome/messenger/migration/migration.dtd

```diff
@@ -16,28 +16,28 @@
 <!ENTITY importFromNothing.label        "Non importare nulla">
 <!ENTITY importFromNothing.accesskey    "m">
 
-<!ENTITY welcome.title                  "Welcome to &brandShortName;">
-<!ENTITY tagline.label                  "A Better Way To Do Email">
-<!ENTITY thanks.label                   "Thanks for trying out &brandShortName;!">
-<!ENTITY quickStart1.label              "See the ">
-<!ENTITY quickStart2.label              "Quickstart Guide"> 
-<!ENTITY quickStart3.label              " to learn about all the features.">
-<!ENTITY start.label                    "Start using &brandShortName;">
-    
-<!ENTITY migration.title                "Account Migration Wizard">
-<!ENTITY migrationMac.title             "Account Migration Assistant">
-<!ENTITY importOptions.title            "Import Options">
+<!ENTITY welcome.title                  "Benvenuto a &brandShortName;">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY welcome.title "Benvenuto su &brandShortName;">